### PR TITLE
Diagnostics

### PR DESCRIPTION
**Pull request by jonmathews-ensoft**

### CODE REVIEW

File: edu.iastate.flowminer/src/edu/iastate/flowminer/actions/FlowMinerAction.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ String chooseFile(String message, String typeDescription, String typeExtension){
 	}
 	
 	String chooseSummaryFile(String message){
-		return chooseFile(message, "FlowMiner Summaries", ".xml.gz");
+		return chooseFile(message, "FlowMiner Summaries", ".xml-gz");
```
Comment (powerofpi) [inline edu.iastate.flowminer/src/edu/iastate/flowminer/actions/FlowMinerAction.java]: I'm curious why this change?

**Comment by powerofpi** [comment]
Other than my question, this looks good. Thanks Jon.

**Comment by jonmathews-ensoft** [comment]
I don't recall, but it's possible there was some awkwardness in getting the
summary file to show up in the dialog on macOS.  You can revert anything
that doesn't immediately work and I'll redo it if necessary.
